doc/userguide/rules/ftp-keywords.rst

```diff
@@ -71,3 +71,36 @@ Examples of commands are:
 * PASV
 * RETR
 
+
+ftp.reply
+---------
+
+This keyword matches on a FTP reply string. Note that there may be multiple reply strings for
+an FTP command. ``ftp.reply`` is a sticky buffer and can be used as a fast pattern. Do not
+include the completion code.
```

Comment:
Suggestion (if I understood this correctly)
```suggestion
an FTP command. ``ftp.reply`` is a sticky buffer and can be used as a fast pattern. Do not
include the completion code in the `content` to match upon (see examples).
```

Reply:
Thanks